<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <exportmacro>SCHNAPPS_PLUGIN_SURFACE_MODELISATION_API</exportmacro>
 <class>Filtering_Dialog</class>
 <widget class="QDialog" name="Filtering_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filtering</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QListWidget" name="list_maps"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Position attribute :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_positionAttribute">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Normal attribute :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_normalAttribute">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="3">
      <widget class="QPushButton" name="button_bilateral">
       <property name="text">
        <string>Bilateral</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="button_average">
       <property name="text">
        <string>Average</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="button_taubin">
       <property name="text">
        <string>Taubin</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="button_laplacian">
       <property name="text">
        <string>Laplacian</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>list_maps</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label_2</zorder>
  <zorder></zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
